coderd/database/queries/prebuilds.sql

```diff
@@ -48,6 +48,56 @@ WHERE tvp.desired_instances IS NOT NULL -- Consider only presets that have a pre
   -- AND NOT t.deleted -- We don't exclude deleted templates because there's no constraint in the DB preventing a soft deletion on a template while workspaces are running.
 	AND (t.id = sqlc.narg('template_id')::uuid OR sqlc.narg('template_id') IS NULL);
 
+-- name: GetRunningPrebuiltWorkspacesOptimized :many
+WITH latest_prebuilds AS (
```

Comment:
nit: adding a comment for all CTEs for readability, e.g., 
```suggestion
-- Returns a list of workspaces that meet all the following:
--   * The latest build transition was 'start'.
--   * The latest build job status is 'succeeded'.
--   * The workspace is a prebuild (prebuilds system user id).
--   * The workspace is not marked as deleted.
WITH latest_prebuilds AS (
```